### ISSUE TITLE
Import and export FDS files

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpseditor#128]
In Gitlab by @gjaeger on Jan 25, 2019, 11:13 [[origin](https://gitlab.version.fz-juelich.de/jupedsim/jpseditor/issues/115)]

FDS (_Fire Dynamics Simulator_) is a beautiful tool, we should also be able to import and export fds-files.

**Comment by sainho93** [comment]
@chraibi should JPSeditor try to collaborate with FDS still, or drop this issue?

**Comment by gjaeger** [comment]
I still like an import-export function for FDS files. 

I am thinking of using jpscore/jpsfire in fire and evacuation simulations. jpsreport could also be used for evaluating calculations with other programs. An Import/Export function would be a first step.

**Comment by sainho93** [comment]
I agree it's a great feature, but it's also a different development path. If there is no strong requirement for our research project, I think this issue isn't in the high priority level.
